nvtabular/dask/io.py

```diff
@@ -159,17 +162,28 @@ class DaskDataset:
         to GPU memory capacity). Ignored if part_size is passed
         directly. Note that the underlying engine may allow other
         custom kwargs to override this argument.
+    storage_options: None or dict
```

Comment:
@benfred - Note that I am explicitly adding `storage_options` here to hopefully help with future plans to support [remote data](https://docs.dask.org/en/latest/remote-data-services.html).

Reply:
nice! thanks for the link